.gitignore

```diff
@@ -25,4 +25,8 @@ buildNumber.properties
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
+# eclipse
```

Comment:
Can you please put IDE specific settings in your own global `.gitignore`, see https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
Removed.

Reply:
I am not sure if these tests were passing before.

Reply:
Okay fair enough I will fix them later, I will merge and publish tomorrow then